components/BlurhashImage.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div>
+    <transition-group
+      mode="in-out"
+      name="fade"
+      style="
+        height: 100%;
+        width: 100%;
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+      "
+    >
+      <blurhash-canvas
+        v-if="item.ImageBlurHashes.Primary && item.ImageTags.Primary"
+        key="canvas"
+        :hash="item.ImageBlurHashes.Primary[item.ImageTags.Primary]"
+        :width="width"
+        :height="height"
+        :punch="punch"
+        style="
+          height: 100%;
+          width: 100%;
+          position: absolute;
+          top: 0;
+          left: 0;
+          right: 0;
+          bottom: 0;
+        "
+      />
+      <v-img
+        v-if="item.ImageTags.Primary"
+        key="image"
+        style="
+          height: 100%;
+          width: 100%;
+          position: absolute;
+          top: 0;
+          left: 0;
+          right: 0;
+          bottom: 0;
+        "
+        :src="imageLink(item.Id)"
+        v-bind="$attrs"
+      />
+    </transition-group>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { BaseItemDto } from '~/api';
+
+export default Vue.extend({
+  props: {
+    item: {
+      type: Object as () => BaseItemDto,
+      required: true
+    },
+    width: {
+      type: Number,
+      default: 32
+    },
+    height: {
+      type: Number,
+      default: 32
+    },
+    punch: {
+      type: Number,
+      default: 1
+    }
+  },
+  methods: {
+    imageLink(id: string) {
+      return `${this.$axios.defaults.baseURL}/Items/${id}/Images/Primary`;
```

Comment:
Probably better to use imageHelper

Reply:
Yeah, this is not quite done yet, I need to update it for the recent merges (including the proper image mixin) and fix the weird TypeScript issues.

Reply:
Alright, should be good to go now :+1: